data/prixa-tech.json

```
@@ -0,0 +1,16 @@
+{
+ "name": "Prixa Technologies",
+ "establishedAt": 1446014709,
+ "location": "Jawalakhel, Lalitpur, Nepal",
+ "website": "https://prixa.net",
+ "email": "info@prixa.org",
+ "phone": "9801970966",
+ "socials": {
+   "facebook": "https://www.facebook.com/prixatech",
+   "github": null,
+   "instagram": null,
+   "linkedin": "https://www.linkedin.com/company/prixa",
+   "twitter": "https://twitter.com/prixatech",
+   "youtube": null

```

```suggestion
   "youtube": "https://www.youtube.com/channel/UCEKpdNQryb_-kqfmolDknAw"

```
This seems like their youtube channel.